connectors/src/connectors/zendesk/lib/sync_ticket.ts

```diff
@@ -185,8 +185,9 @@ ${comments
       );
       author = null;
     }
+    const createdAt = new Date(Number(comment?.created_at));
     return `
-[${new Date(Number(comment?.created_at)).toISOString()}] ${
+[${isNaN(createdAt.getTime()) ? "Unknown date" : createdAt.toISOString()}] ${
```

Comment:
FMI how / when does this happen ? What does created_at look like in such cases ? Can't we just handle `undefined` ?

Reply:
don't really know, I got a bit lazy there and didn't want to log/reproduce and then have another go

Reply:
actually I'll remove all of that, `created_at` is already supposed to be in ISO format 😇 
https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_import/#ticket-import